powersimdata/data_access/fs_helper.py

```diff
@@ -0,0 +1,57 @@
+import fs
+from fs.multifs import MultiFS
+
+from powersimdata.data_access.ssh_fs import WrapSSHFS
+from powersimdata.utility import server_setup
+
+
+def get_blob_fs(container):
```

Comment:
Missing docstring

Reply:
Added these and squashed into the same commit